services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcBaseSpecification.groovy

```diff
@@ -6,18 +6,15 @@ import org.openkilda.functionaltests.BaseSpecification
 import org.openkilda.functionaltests.extension.tags.Tags
 import org.openkilda.messaging.info.event.SwitchInfoData
 
-import spock.lang.Shared
+import groovy.transform.Memoized
 
 @Tags(HARDWARE)
-class GrpcBaseSpecification extends BaseSpecification{
-    @Shared
-    String switchIp
-    @Shared
-    SwitchInfoData nFlowSwitch
-
-    def setupOnce() {
-        requireProfiles("hardware")
-        nFlowSwitch = northbound.activeSwitches.find { it.description =~ /NW[0-9]+.[0-9].[0-9]/ }
-        switchIp = nFlowSwitch.address
+class GrpcBaseSpecification extends BaseSpecification {
+    @Memoized
+    List<SwitchInfoData> getNoviflowSwitches() {
+        northbound.activeSwitches.findAll {
+            def matcher = it.description =~ /NW[0-9]+.([0-9].[0-9])/
+            return matcher && matcher[0][1] > "6.4"
```

Comment:
a brief comment of why we pick only 6.4+ versions would be nice

Reply:
added
`// it is not working properly if version <= 6.4`